packages/edit-post/src/plugins/index.js

```diff
@@ -29,6 +30,7 @@ registerPlugin( 'edit-post', {
 								{ __( 'Manage All Reusable Blocks' ) }
 							</MenuItem>
 							<KeyboardShortcutsHelpMenuItem onSelect={ onClose } />
+							<WelcomeGuideMenuItem onSelect={ onClose } />
```

Comment:
not specific to this PR but we're using the "More Menu" as a place where we just add things we're not sure where to add. Some design thinking here could be good. cc @jasmussen @mapk

Reply:
`onSelect` can be omitted, it's not used and probably shouldn't be so keyboard users could experience the focus moved back to this item when they close the welcome guide.

Reply:
Agree that design input would be good.

I think ideally a button like this would go into a _Help_ menu. For example, Figma has this nice floating _Help_ button in the bottom right of the screen:

<img width="180" alt="Screen Shot 2019-11-28 at 11 58 02" src="https://user-images.githubusercontent.com/612155/69800976-9b09b680-11d6-11ea-805f-8d989280fd7e.png">

Until we have something like that, though, I think it makes sense to put this in the _More_ menu.

---

@gziolo: Good catch! Fixed in 588c2d53b.